encrypted-media-respec.html

```diff
@@ -3172,7 +3213,7 @@
 
         <section id="allow-identifiers-cleared">
           <h3>Allow Identifiers to Be Cleared</h3>
-          <p>As a consequence of the requirements in <a href="#allow-persistent-data-cleared">Allow Persistent Data to Be Cleared</a>, implementations that use <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> MUST allow the user to clear those identifiers such that they are no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
+          <p>As a consequence of the requirements in <a href="#allow-persistent-data-cleared">Allow Persistent Data to Be Cleared</a>, implementations that <a href="#uses-distinctive-identifiers-or-distinctive-permanent-identifiers">Distinctive Identifier(s) or Distinctive Permanent Identifier(s)</a> MUST allow the user to clear the <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> and values associated with <a def-id="distinctive-permanent-identifier-maybe-plural"></a> such that they are no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
```

Comment:
It's okay to adjust naming later.  I still believe though that it's confusing to apply "Distrinctive" to the Permanent Identifiers name when many of the Distinctive requirements don't apply to it.